### ISSUE TITLE
[Bug Report] v-stepper-step[editable] not accessible with keyboard

### ISSUE THREAD

**Issue by WilcoFiers** [vuetifyjs/vuetify#9975]
### Environment
**Vuetify Version:** 2.1.14
**Vue Version:** 2.6.10
**Browsers:** Chrome 79.0.3945.79
**OS:** Mac OS 10.15.1

### Steps to reproduce
1. Open the codepan
2. Use tab navigation to move through the page. Notice that Step 1 and Step 2 are never focused.
3. Click on Step 2, then on Step 1 using the mouse pointer. Notice these two steps are interactive

### Expected Behavior
When given the "editable" property, v-stepper-focus should be focusable, have a role="button" attribute, and and respond to pressing enter or space.

Also noticed that even when tabindex is added to v-stepper-step, @click events can still not be fired using keyboard.

### Actual Behavior
The v-stepper-step with an editable attribute is not focusable, has no role, and when these are added, @click can still not be fired from it.

### Reproduction Link
<a href="https://codepen.io/wilcofiers/full/zYxKXpg" target="_blank">https://codepen.io/wilcofiers/full/zYxKXpg</a>

**Comment by dos077** [comment]
Hmm, the labels are just vanilla divs. I can think of two ways to make them keyboard navable.
1. Wrap the label with an A tag
2. The hard way, add tabindex to the div and a key listener for enter key when element is in focus

**Comment by WilcoFiers** [comment]
Yeah, either would work.

**Comment by phiter** [comment]
Simply adding `tabindex` and/or `role="button"` to the div won't work. I believe the browser only considers enter and space as click events in buttons and anchors.

We have to think of a better solution for this.
